cla_backend/apps/diagnosis/management/commands/delete_public_diagnosis.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import datetime
+from django.core.management.base import NoArgsCommand
```

Comment:
PEP8 - there should be a blank line between core library imports and 3rd party imports